pymeasure/instruments/keysight/keysightB2902B.py

```diff
@@ -0,0 +1,47 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2025 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+from pymeasure.instruments import Instrument
+from .keysightB2901B import KeysightB2901B, SMUChannel
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class KeysightB2902B(KeysightB2901B):
+    """
+    This represents the Keysight B2902B Source-Measure Unit interface.
+
+    .. code-block:: python
+
+        smu = KeysightB2902B(address)
+
+    """
+
+    def __init__(self, adapter, name="Keysight B2902B Source-Measure Unit", **kwargs):
+        super().__init__(
+            adapter, name, **kwargs)
+
+    channel_2 = Instrument.ChannelCreator(SMUChannel, 2)
```

Comment:
If this instrument code is such small, you may add it directly to the main class (2901B), which you may rename to something like 29xxBSeries.